forcedotcom/salesforcedx-vscode
# fix: remove copying of apex.db to workspace on LS startup

### What does this PR do?

Removes the initialization of apex.db each time the Apex LS is started
Language server will be responsible for "installing"/"upgrading" apex.db

fix: disable file watcher for apex std classes dir

### What issues does this PR fix or reference?
@W-13593866@
@W-13739509@

QA Notes:

- Start vsce in a project that has an apex.db in `.sfdx/tools` dir and the file should not be overlayed by extension
- Start vsce in a project that does not have an apex.db in `.sfdx/tools` dir and the file should be created by the language server
